<launch>
    <!-- <node pkg="trajectory_pkg" type="object_detector.py" name="object_detector" output="screen"/>

    <node pkg="trajectory_pkg" type="ur5_pick_and_place.py" name="ur5_pick_and_place" output="screen"/> -->

    <group ns="objetos">
      <!-- add box blue -->
      <node name="spawn_box_blue" pkg="gazebo_ros" type="spawn_model" output="screen"
        args="-file $(find trajectory_pkg)/models/pick_place/bin_blue/model.sdf -sdf -model box_blue -x -0.8 -y 0.240476 -z 1.025" />

      <!-- add box green -->    
      <node name="spawn_box_green" pkg="gazebo_ros" type="spawn_model" output="screen"
        args="-file $(find trajectory_pkg)/models/pick_place/bin_green/model.sdf -sdf -model box_green -x -0.8 -y -0.009524 -z 1.025" />

      <!-- add box red -->    
      <node name="spawn_box_red" pkg="gazebo_ros" type="spawn_model" output="screen"
        args="-file $(find trajectory_pkg)/models/pick_place/bin_red/model.sdf -sdf -model box_red -x -0.8 -y -0.259524 -z 1.025" />

      <!-- add block blue 1 -->    
      <node name="spawn_block_blue_1" pkg="gazebo_ros" type="spawn_model" output="screen"
        args="-file $(find trajectory_pkg)/models/pick_place/block_blue/model.sdf -sdf -model block_blue_1 -x -0.427131 -y 0.003185 -z 1.045" />

      <!-- add block green 1 -->    
      <node name="spawn_block_green_1" pkg="gazebo_ros" type="spawn_model" output="screen"
        args="-file $(find trajectory_pkg)/models/pick_place/block_green/model.sdf -sdf -model block_green_1 -x -0.560503 -y -0.221624 -z 1.045" />

      <!-- add block red 1 -->    
      <node name="spawn_block_red_1" pkg="gazebo_ros" type="spawn_model" output="screen"
        args="-file $(find trajectory_pkg)/models/pick_place/block_red/model.sdf -sdf -model block_red_1 -x -0.571972 -y 0.200934 -z 1.045" />

      <!-- add block blue 2 -->    
      <node name="spawn_block_blue_2" pkg="gazebo_ros" type="spawn_model" output="screen"
        args="-file $(find trajectory_pkg)/models/pick_place/block_blue/model.sdf -sdf -model block_blue_2 -x -0.424412 -y -0.143902 -z 1.045" />

      <!-- add block green 2 -->    
      <node name="spawn_block_green_2" pkg="gazebo_ros" type="spawn_model" output="screen"
        args="-file $(find trajectory_pkg)/models/pick_place/block_green/model.sdf -sdf -model block_green_2 -x -0.441868 -y 0.201144 -z 1.045" />

      <!-- add block red 2 -->    
      <node name="spawn_block_red_2" pkg="gazebo_ros" type="spawn_model" output="screen"
        args="-file $(find trajectory_pkg)/models/pick_place/block_red/model.sdf -sdf -model block_red_2 -x -0.564329 -y 0.006137 -z 1.045" />

    </group>
</launch>